### PR TITLE
cleaned up starter code in dsModel.js and dsRouter.js, added placehol…

### PR DESCRIPTION
**Pull request by virginia-d90**
…ders

## Bug description
Not a bug just tidying up starter code to remove unwanted functions.

## Solution description
Removed unwanted functions

## Which file changed
Clarify which file changed, for example “Added Hello World to index.js”

## Covered unit test cases
Were unit test cases recorded for this fix?

## Tested this issue fix on all browser
[x] Chrome
[x] Safari
[x] Internet Explorer
[x] Firefox

## Keep Pull Request Simple
2 file changed

### CODE REVIEW

File: api/dsService/dsRouter.js [hunk 1]
```diff
@@ -3,118 +3,13 @@ const router = express.Router();
...
-  dsModel
-    .getPrediction(x1, x2, x3)
-    .then((response) => {
-      res.status(200).json(response.data);
-    })
-    .catch((error) => {
-      console.error(error);
-      res.status(500).json(error);
-    });
+//Placeholder endpoints to be built:
```
Comment (caw442000) [inline api/dsService/dsRouter.js]: Comment not needed for this code

**Review by caw442000 — CHANGES_REQUESTED** [review]
No comments needed on empty functions

File: api/dsService/dsRouter.js (hunk 1 above)
Comment (virginia-d90) [reply api/dsService/dsRouter.js]: changes made

**Review by caw442000 — APPROVED** [review]

**Review by caw442000 — APPROVED** [review]
Great start to cleanup

**Review by Juan-Rivera — APPROVED** [review]

**Pull request merged**